src/components/pill-selector-dropdown/PillSelector.scss

```diff
@@ -122,6 +122,10 @@
             line-height: $bdl-line-height + $bdl-grid-unit * 2;
         }
     }
+
+    .SelectorDropdown-overlay > ul.overlay {
```

Comment:
any particular reason for this extra specificity?

Reply:
There can be a case where a descendent can be an overlay. This is why I needed to specify this rule